config/common/regiment_types.cwt

```diff
@@ -4,15 +4,108 @@ types = {
     }
 }
 
-alias[modifier:<regiment_type>_max_size_add] = int_value_field
-alias[modifier:<regiment_type>_max_size_mult] = value_field
-alias[modifier:<regiment_type>_siege_value_add] = int_value_field
-alias[modifier:<regiment_type>_siege_value_mult] = value_field
-alias[modifier:<regiment_type>_damage_add] = int_value_field
-alias[modifier:<regiment_type>_damage_mult] = value_field
-alias[modifier:<regiment_type>_toughness_add] = int_value_field
-alias[modifier:<regiment_type>_toughness_mult] = value_field
-alias[modifier:<regiment_type>_pursuit_add] = int_value_field
-alias[modifier:<regiment_type>_pursuit_mult] = value_field
-alias[modifier:<regiment_type>_screen_add] = int_value_field
-alias[modifier:<regiment_type>_screen_mult] = value_field
\ No newline at end of file
+regiment_type = {
+    ### Unit archetype of the regiment. Used for modifiers and counters.
+    type = value_set[unit_type]
+    ## cardinality = 0..1
+    ### Optional, use this name instead of the key when selecting icon graphics.
+    icon = scalar
+
+    ## cardinality = 0..1
+    ### If set, can't be recruited. For use by mercs/holy orders instead. Defaults to yes.
+    can_recruit = bool
+    ## cardinality = 0..1
+    ### Whether it's allowed to have this regiment type for mercenaries and holy orders. Defaults to yes.
+    allowed_in_hired_troops = bool
+    # TODO: IDK what these can do exactly.
```

Comment:
I _believe_ but haven't tested this but I think setting these bools to true allows them to be selected in mercenary companies/holy orders if there are no valid options defined for the relevant company/order after filtering out regiments that haven't been unlocked.